recipes/seadex-essentials/all/conanfile.py

```diff
@@ -48,14 +48,12 @@ def config_options(self):
     def configure(self):
         if self.options.shared:
             self.options.rm_safe("fPIC")
-        self.options["fmt/*"].header_only = True
-        self.options["spdlog/*"].header_only = True
```

Comment:
This is a bad idea, because now it doesn't build out of the box, and c3i doesn't test this recipe anymore (all configurations raise Invalid Configuration)